package.json

```diff
@@ -28,7 +28,7 @@
     "upgrade-deps:all": "npm-check --update --save-exact",
     "upgrade-deps:production": "npm-check --update --save-exact --production",
     "upgrade-deps:dev": "npm-check --update --save-exact --dev-only",
-    "test:e2e": "wdio run ./wdio.conf.js"
+    "test:e2e": "wdio run ./tests/e2e/config/wdio.conf.js"
```

Comment:
For consistency, i think we should leave the `wdio.conf.js` at the root, like all the other configuration files, unless you have a reason that im unaware of to put it here!